plugins/modules/luks_device.py

```diff
@@ -79,6 +79,16 @@
               value is a string with the passphrase."
         type: str
         version_added: '1.0.0'
+    keyslot:
+        description:
+            - "Adds the O(keyfile) or O(passphrase) to a specific keyslot when
+              creating a new container on O(device). Parameter value is the
+              number of the keyslot."
+            - "NOTE that a device of O(type) luks1 supports the keyslot numbers
+              0-7 and a device of O(type) luks2 supports the keyslot numbers
+              0-32. In order to use the keyslots 8-31 when creating a new
+              container, setting O(type) to luks2 is required."
+        type: int
```

Comment:
```suggestion
        type: int
        version_added: 2.16.0
```
Same for the other new options.